README.md

```diff
@@ -54,12 +52,9 @@ About
 LDK/`rust-lightning` is a generic library which allows you to build a Lightning
```

Comment:
I'd change this from:

"LDK/`rust-lightning` is a generic library which allows... exactly correct." 

to:

"LDK/`rust-lightning` is a library that allows... correct."